migrations/versions/170_add_login_tracking.py

```diff
@@ -0,0 +1,34 @@
+"""Add fields for tracking logins
+
+Track nuber of failed logins so that we can automatically lock accounts
+when they fail to log in too many times.
+Track the time the last logged in so that we can remove old users.
+
+Revision ID: 170_add_login_tracking
+Revises: 160_add_framework_slug
+Create Date: 2015-06-30 16:38:13.254662
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '170_add_login_tracking'
+down_revision = '160_add_framework_slug'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.add_column('users', sa.Column('failed_login_count', sa.Integer(), nullable=True))
```

Comment:
`failed_login_count` is not nullable in the model, so probably shouldn't be nullable in the database either.